app/assets/stylesheets/legislation_process.scss

```diff
@@ -986,3 +987,38 @@
     transform: translateX(-50%);
   }
 }
+
+// 10. Legislation draft table
+// -------------------------
+
+.draft-text {
+  table {
+    display: block;
+    width: 100%;
+    overflow-x: auto;
+    border: $table-border;
+    border-collapse: collapse;
+    color: $body-font-color;
+    background-color: $table-background;
+
+    th {
+      background: $table-head-background;
+      color: $table-head-font-color;
+    }
+
+    tr {
+
+      td {
+        line-height: $line-height;
+      }
+
+      &:nth-child(odd) {
+        background: $table-background;
+      }
+
+      &:nth-child(even) {
+        background: $table-striped-background;
+      }
+    }
+  }
+}
```

Comment:
I see that you have used the variables available for the tables. But currently in consul for the tables that appear in the public part the styles from `/assets/stylesheets/layout.scss:1203` are being applied.

What do you think if we remove these styles that have been added and then it will render the same as the rest of the tables?

What might be a good idea is to add the scroll. I don't think it would be necessary to add the new section // 10. Legislation draft tables and we can add the following code on `legislation_process.scss:503` inside the `.draft-text` class:

```
table {
  @include table-scroll;
}
```